ui/src/app/index/login.component.html

```diff
@@ -29,8 +29,12 @@
                   </ion-item>
                   <ion-item>
                     <ion-label position="floating" translate>Login.passwordLabel</ion-label>
-                    <ion-input #password required type="password" [placeholder]="'Login.passwordLabel' | translate"
+                    <ion-input #password required [type]="showPassword ? 'text' : 'password'" [placeholder]="'Login.passwordLabel' | translate"
                       value="user"></ion-input>
+                    <ion-button slot="end" fill="clear" (click)="togglePasswordVisibility()"
+                        [attr.aria-label]="(showPassword ? 'Login.hidePassword' : 'Login.showPassword') | translate">
+                        <ion-icon [name]="showPassword ? 'eye-off-outline' : 'eye-outline'"></ion-icon>
+                    </ion-button>
```

Comment:
please create component for this ion-input paired with ion-button, 

btw are you familiar with this: https://ionicframework.com/docs/api/input-password-toggle
would this be suitable as well

Reply:
i would use the ionic component if possible

Reply:
Unfortunately, ion-input-password-toggle is avaiable from ionic 8! https://github.com/ionic-team/ionic-framework/releases/tag/v8.0.0-beta.4